client/src/components/Button/Button.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const Button = (props) => {
```

Comment:
does this expect children? if it does might be best to implement it now

Reply:
Let's discuss this tomorrow during standup.